sign/pairing-uri.md

```diff
@@ -20,8 +20,11 @@
 ### Optional
 
 - `relay-data` (STRING) = hex data payload used for relay
+- `sdk` (STRING) = sdk that should handle this uri
 
-## Example
+## Examples
+
+1. Only required parameters
```

Comment:
maybe we call this the deprecated example since we want to move forward with the sdk parameter

Reply:
Nice call, I like it

Reply:
Also reordered them